README.md

```diff
@@ -1,22 +1,48 @@
 2023/03/01
 Hsiang-He Lee (lee1061@llnl.gov)
+2023/05/24 updated
 
-E3SM chemistry diagnostic package version1.0 (ChemDyg v1.0) is designed for E3SM chemistry development. There are total 11 types of plot and 3 types of table for model-to-model and model-to-observation comparison.
+E3SM chemistry diagnostic package version 0.1.0 (ChemDyg v0.1.0) is designed for E3SM chemistry development. There are total 11 types of plot and 3 types of table for model-to-model and model-to-observation comparison.
 
 Index page and example figures:
 https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.lee1061/20220914.PAN.MZThet.v2.LR.bi-grid.amip.chemUCI_Linozv3/e3sm_chem_diags/plots/
 
 ChemDyg is executed by zppy, which is a post-processing toolchain for E3SM written in Python (https://github.com/E3SM-Project/zppy).
 
-*How to run ChemDyg via zppy script on chrysalis?
-1. Setup environment 
->source /gpfs/fs1/soft/chrysalis/manual/anaconda3/2020.11/etc/profile.d/conda.sh
->conda activate zppy_dev_chem
+# How to run ChemDyg via zppy script on chrysalis?
```

Comment:
It would be ideal if you could provide more general instructions for all zppy-supported machines, not just Chrysalis.  This will presumably be via E3SM-Unified.  I guess it's hard to guess exactly what these will be for now.

Reply:
At this stage, I really hard to imagine what will be in E3SM-Unified. If a general instruction would be better, how about this (I am not sure if the processes make sense or not)
How to run ChemDyg via zppy script on chrysalis (same as other zppy-supported machines)?

source /lcrc/soft/climate/e3sm-unified/load_latest_e3sm_unified_chrysalis.sh
*Download zppy
git clone git@github.com:E3SM-Project/zppy.git
cd zppy
*Install chemdyg package
conda install -y -c chemdyg
*Install zppy in environment
python -m pip install .
*Once this is done, you call is as
zppy -c ChemDyg_example_script.cfg

Reply:
zppy will be in E3SM-Unified so folks shouldn't need to do all that. They should just need to do:
```
source /lcrc/soft/climate/e3sm-unified/load_latest_e3sm_unified_chrysalis.sh
wget https://raw.githubusercontent.com/E3SM-Project/ChemDyg/main/ChemDyg_example_script.cfg
zppy -c ChemDyg_example_script.cfg
```
You should provide them with the E3SM-Unified paths for all supported machines here in the README, as zppy does in its docs:
https://e3sm-project.github.io/zppy/_build/html/main/getting_started.html#activate-e3sm-unified-environment